data/remnant/remnant 2 cognizance.txt

```diff
@@ -1071,8 +1081,10 @@ mission "Remnant: Cognizance 24"
 	destination "Ssil Vida"
 	cargo "outpost equipment" 11
 	passengers 1
+	blocked "You receive a message from Dusk and Taely indicating the next mission requires space for 11 tons of cargo and a passenger."
```

Comment:
```suggestion
	blocked "You receive a message from Dusk and Taely indicating the next mission requires space for <cargo> and <passengers>."
```
Does this help?

Reply:
First, just so you know @tibetiroka , by using `<cargo>` it will output `<x> ton(s) of <commodity>`, while but by just using `<tons>` it will output `<x> ton(s)` alone.

And secondly, `<passengers>` will only output either "passenger" or "passengers", depending on if there're more than 1 passenger. While `<bunks>` will output the number of passengers alone.

Reply:
```suggestion
	blocked "You receive a message from Dusk and Taely indicating the next mission requires space for <tons> of cargo and <fare>."
```